ParraFeedback/Views/Modals/Fields/ParraFeedbackFormTextFieldView.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  ParraFeedbackFormTextFieldView.swift
+//  ParraFeedback
+//
+//  Created by Mick MacCallum on 6/11/23.
+//  Copyright © 2023 Parra, Inc. All rights reserved.
+//
+
+import Foundation
+import ParraCore
+import SwiftUI
+import Combine
+
+struct ParraFeedbackFormTextFieldView: ParraFeedbackFormFieldView {
+    typealias FieldData = FeedbackFormTextFieldData
+
+    @State var formId: String
+    @State var field: FeedbackFormField
+    @State var fieldData: FieldData
+    @State var onFieldDataChanged: ParraFeedbackFormFieldUpdateHandler
+
+    @State private var text = ""
+    @State private var hasReceivedInput = false
+
+    var body: some View {
+        let errorMessage = validate(
+            text: text.trimmingCharacters(in: .whitespacesAndNewlines),
+            data: fieldData,
+            requiredField: field.required ?? false
+        )
+
+        VStack(spacing: 5) {
+            let textEditor = TextEditor(text: $text)
+                .frame(minHeight: 100, idealHeight: 100, maxHeight: 240)
+                .padding()
+                .background(.white)
+                .clipShape(RoundedRectangle(cornerRadius: 8))
+                .background(
+                    RoundedRectangle(cornerRadius: 8)
+                        .stroke(Color.gray, lineWidth: 1)
+                )
+                .onReceive(Just(text)) { newValue in
+                    onFieldDataChanged(field.name, newValue, errorMessage == nil)
+                }
+                .onChange(of: text) { newValue in
+                    hasReceivedInput = true
+                }
+            if #available(iOS 16.0, *) {
```

Comment:
They really need a clean syntax for available here so you don't have to break it up lol